src/components/DishDetailComponent.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import { Card, CardImg, CardText, CardBody, CardTitle } from 'reactstrap';
+
+class DishDetail extends Component {
+    formatDate(date) {
+        // formats like Jan XX, XXXX
+        return new Date(date)
+                 .toLocaleDateString('en-EN', {weekday: undefined, month: 'short', day: 'numeric', year: 'numeric' })
```

Comment:
I think all attributes on objects default `undefined` if you don't assign them a value, so you do not need to set weekday to `undefined` here :).

Also this is more of a nit, but I think you should move this object to a variable w/ a canonical name:

```
// formats like Jan XX, XXXX
const MonthName-Day-Year-DateFormat = {month: 'short', day: 'numeric', year: 'numeric' };
...
return new Date(date).toLocaleDateString('en-EN', MonthName-Day-Year-DateFormat)

```

But I find that's not really important at all unless you use something a few places.